docs/commands.md

```diff
@@ -150,8 +141,6 @@
     * Shows you your mcmmo stats
   * /tags
     * Opens the tags GUI to select a tag
-  * /titan pack
```

Comment:
`/titan pack` still exists, can you re-add this?

Reply:
![image](https://github.com/user-attachments/assets/90af2df9-9f91-4885-ba0c-20012907cf19)
It doesn't show up for me

Reply:
okay, this is fixed. can you re-add this now.

Reply:
its now added back should be able to merge unless any other things need a change